utils/scripts/install-centos8.sh

```diff
@@ -67,5 +68,20 @@ dnf install \
     python3-yaml \
     sg3_utils \
     valgrind-devel \
+    which \
     yasm
+
+echo "is fedora java? $fedora_java"
+
+if [ "$fedora_java" = "yes" ]; then
+	dnf install java-1.8.0-openjdk-devel
+	curl --output ./apache-maven-3.6.3-bin.tar.gz \
+https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz \
+&& tar -xf apache-maven-3.6.3-bin.tar.gz \
+&& find ./ -name "mvn" | \
+{ read -r d; f="$(dirname "$d")"; cp -r "$(readlink -f "$f")"/../* /usr/local; }
```

Comment:
Why are we doing this for Fedora but using the `maven` RPM for other distros?

Reply:
see above comments